Improved the PKGBUILD according to our standard

Antiz96 opened the pull request:
Hi,

A round of improvements for the `jade` PKGBUILD:

- Switched all vars to the `${var}` format.
- Reordered the top vars to match the Arch PKGBUILD template order.
- Switch the `license` from `GPLv3` to `GPL3` (it has to be named that way).
- Deleted the extra `GPL` license (if there's multiple licenses, they have to both go into the same `license` array).
- Switched source from `git` to release's archive. Added the integrity check accordingly.
- Modified the `package ()` function to use `install` instead of `mkdir + chmod + install`.

What do you think?

SomethingGeneric reviewed (approved):
LGTM! Thanks for the awesome work, as usual.

FabioLolix commented:
Good

FabioLolix commented:
Maybe you want to revist build() and add prepare() as for rust guidelines, like in malachite pkgbuild for example

```
prepare() {
    cd "${srcdir}/${pkgname}-${pkgver}"
    cargo fetch --locked --target "${CARCH}-unknown-linux-gnu"
}

build() {
    cd "${srcdir}/${pkgname}-${pkgver}"
    export RUSTUP_TOOLCHAIN=stable
    export CARGO_TARGET_DIR=target
    cargo build --frozen --release --all-features
}
```

SomethingGeneric commented on PKGBUILD:
```diff
@@ -1,25 +1,24 @@
+source=("${pkgname}-${pkgver}::${url}/archive/v${pkgver}.tar.gz")
+sha256sums=('fcf1b168e14c0903572d45b9a16a241a2b72edf2f496745502c7edf24ba3ee62')
 
 build() {
```
Let's adapt as @FabioLolix suggests, especially since Malachite follows that already

Antiz96 replied:
Done in latest commit

SomethingGeneric commented on PKGBUILD:
```diff
@@ -1,25 +1,24 @@
     cargo build --release
 }
 
 package() {
```
^

Antiz96 replied:
Done in latest commit

Antiz96 commented:
I added the `prepare ()` function and I modified the `build ()` function according to rust guidelines, as suggested by @FabioLolix.
Build was successful.
